deploy/kubernetes/base/node_linux/node.yaml

```diff
@@ -38,13 +38,13 @@ spec:
               mountPath: /csi
             - name: registration-dir
               mountPath: /registration
-          livenessProbe:
```

Comment:
I think we would want to leave this liveness probe, to check that registration is still good (the CSI liveness probe won't know about that).

Reply:
the `mode` flag is pretty recent with more background in https://github.com/kubernetes-csi/node-driver-registrar/issues/143, there are two checks that need to be done: checking that node-driver-registrar successfully registered the driver with the kubelet (done with a different running mode of node-driver-registrar) and once the CSI Driver is registered successfully check that the CSI Driver is healthy (done through https://github.com/kubernetes-csi/livenessprobe)

Q: isn't kubernetes-csi/livenessprobe enough?
Unfortunately livenessprobe only restarts the CSI Driver when there might be another issue in node-driver-registrar as described in the issue.